js/src/base-component.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * --------------------------------------------------------------------------
+ * Bootstrap (v5.0.0-alpha3): base-component.js
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/main/LICENSE)
+ * --------------------------------------------------------------------------
+ */
+
+import Data from './dom/data'
+
+class BaseComponent {
+  constructor(element) {
+    if (!element) {
+      return
+    }
+
+    this._element = element
+    Data.setData(element, this.constructor.DATA_KEY, this)
+  }
+
+  dispose() {
+    Data.removeData(this._element, this.constructor.DATA_KEY)
+    this._element = null
+  }
+
+  /** Static */
+
+  static getInstance(element) {
+    return Data.getData(element, this.DATA_KEY)
+  }
+
+  static get DATA_KEY() {
+    return null
+  }
+}
```

Comment:
What if you add a getter here for `VERSION` also and keep the version in the base component because it is same in all the components 🤔
@Johann-S

Reply:
@rohit2sharma95 can you try it and see how it goes? If it works, you can make a PR against this branch :)

Reply:
Indeed adding `VERSION` would be great 👌